apps/dotcms-ui/src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.html

```diff
@@ -72,6 +72,20 @@
             ></dot-field-validation-message>
         </div>
     </div>
+    <div class="p-field push-publish-dialog__timezone-label">
+        <span>{{ localTimezone }}</span> -
+        <a href="#" (click)="toggleTimezonePicker($event)"> {{ changeTimezoneActionLabel }}</a>
```

Comment:
use the `dm` pipe

Reply:
it is not possible `changeTimezoneActionLabel`  changes when link clicked
https://github.com/dotCMS/core-web/blob/2793c9b3a59d65a66273fd149925c78c924e3a99/apps/dotcms-ui/src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.ts#L101